Makefile

```diff
@@ -408,7 +408,10 @@ canary-test: node-canary
 	# TODO: uncomment that!
 	#_assets/scripts/canary_test_mailservers.sh ./config/cli/fleet-eth.prod.json
 
-lint: generate
+lint-panics: generate
+	go run ./cmd/lint-panics -root="$(call sh, pwd)" -skip=./cmd -test=false ./...
```

Comment:
I don't see a `test` flag defined in config :thinking:

Reply:
It's defined by the `go/analysis` package inside `singlechecker.Main` call:
https://github.com/status-im/status-go/blob/f718e5d645f591f62bc4a25cef434664e2b1688c/cmd/lint-panics/main.go#L34

If you run `go run ./cmd/lint-panics --help` you'll get a lot of fancy flags:
```
(venv) [23/10/24 8:24:08] ➜  status-go git:(feat/LogOnPanic-linter) ✗ go run ./cmd/lint-panics -help
logpanics: reports missing defer call to LogOnPanic

Usage: logpanics [-flag] [package]


Flags:
  -V    print version and exit
  -all
        no effect (deprecated)
  -c int
        display offending line with this many lines of context (default -1)
  -cpuprofile string
        write CPU profile to this file
  -debug string
        debug flags, any subset of "fpstv"
  -fix
        apply all suggested fixes
  -flags
        print analyzer flags in JSON
  -json
        emit JSON output
  -memprofile string
        write memory profile to this file
  -root string
        root directory to run gopls (default "/Users/igorsirotin/Repositories/Status/status-go")
  -skip string
        skip paths with this suffix
  -source
        no effect (deprecated)
  -tags string
        no effect (deprecated)
  -test
        indicates whether test files should be analyzed, too (default true)
  -trace string
        write trace log to this file
  -v    no effect (deprecated)
```

These are the standard flags for a go analyzer.